modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIScopeTestCase.java

```diff
@@ -220,10 +313,21 @@ public void destroy() throws Exception {
         }
 
         if (userManagementClient != null) {
-            userManagementClient.deleteUser(USER_JOHN);
-            userManagementClient.deleteRole(SUBSCRIBER_ROLE);
+
+            try {
+                userManagementClient.deleteUser(USER_JOHN);
+                userManagementClient.deleteRole(SUBSCRIBER_ROLE);
+            } catch (RemoteException e) {
+                handleException("Exception occured while removing the artifacts ",e);
```

Comment:
fixed in https://github.com/sambaheerathan/product-apim/commit/40e4b16561d21abf4834ae5ac2b90a1bfc22675e